src/main/drivers/exti.c

```diff
@@ -194,6 +194,14 @@ void EXTIEnable(IO_t io, bool enable)
         EXTI->IMR |= extiLine;
     else
         EXTI->IMR &= ~extiLine;
+#elif defined(STM32H7)
+    uint32_t extiLine = IO_EXTI_Line(io);
+    if (!extiLine)
```

Comment:
Can we have blocks after `if` and `else` here please? Especially these sequences of conditions interspersed with conditionals seem to be asking for regressions to be introduced due to absence of blocks.

Reply:
Good idea.